group_vars/all.yml

```diff
@@ -22,6 +22,7 @@ rust:
 
 solana:
   validator:
+    git_source_url: https://github.com/jito-foundation/jito-solana.git
```

Comment:
are we still using source_url ? we can just keep one right ? and build all the binaries